src/bioregistry/data/bioregistry.json

```diff
@@ -4133,14 +4133,15 @@
   "biorxiv": {
     "description": "The bioRxiv is a preprint server for biology",
     "example": "2021.07.23.453588",
```

Comment:
Side thought: it would be nice to be able to provide multiple examples for situations where there are different subtypes of IDs.

Reply:
I was just thinking about that while I was AFK! I have two ideas on how to do it. I will make a PR to try it out

Reply:
Done in #296